model.py

```diff
@@ -5217,16 +5217,21 @@ def deliverable(self):
         )
 
     @classmethod
-    def with_complaint(cls, _db):
+    def with_complaint(cls, _db, exclude_resolved=True):
```

Comment:
This allows two scenarios: all complaints, and all unresolved complaints. Would it make sense to make this a variable called `resolved`? `resolved=True` means resolved complaints, `resolved=False` means unresolved complaints, `resolved=None` means both. I don't know how often you'd need to look at resolved complaints but right now there's no method for that.